### DISCUSSION TITLE
How do I run Fooocus 2.5.2 from the github files?

### DISCUSSION THREAD

**Discussion by CV133** [lllyasviel/Fooocus#3392] [Q&A] [answered]
I've been using a very early version of fooocus which has worked well for me. However, in the last few days, I'm no longer able to successfully complete the cmd batch file due to recent updates so that the batch file doesn't complete by opening a web page. I've looked into updating and I've downloaded and extracted the latest fooocus 2.5.2 files from the Github site. But!:
Instead of being presented with the expected short list of fooocus batch files to run and start the program, I've only got the main files list that does't indicate what I have to do next to get Fooocus 2.5,2 running successfully. (I should stress here that I'm not a programmer or skilled in the command line interface!) 
Can anybody guide me to get the program running?

**Comment by mashb1t** [comment, marked as answer]
Please follow the instructions in the readme on how to install Fooocus in different ways.
If you're on Windows and have installed Fooocuscusing the 7zip file, you can also follow the instructions in the [v2.5.0 release notes](https://github.com/lllyasviel/Fooocus/releases/tag/v2.5.0) on "how to update". Fooocus then auto-updates on first bootup.

**Reply by CV133** [reply]
Many thanks for responding so speedily. I opened the MD file in notepad and was overwhelmed by the amount of text you've got in there! I did step through it carefully and then came to the part about using run.bat in the files (I have no run.bat in my Fooocus file list!) I did see your subsidiary note about using the CMD box to run: https://github.com/lllyasviel/Fooocus/assets/19834515/c49269c4-c274-4893-b368-047c401cc58c . but it doesn't work! As I said in my opening request, I'm not a 'techie' so can you prehaps give me some clues as to how I can get Fooocus 2.5.2 up and running, perhaps by instructions as to how I can set up a batch file to run Fooocus advanced-realistic. In the meanwhile, I'll step through the rest of the MD file to see if there's any other relevant info. 
Many thanks

**Reply by CV133** [reply]
For your info, I'm running Windows10 on a desktop computer with a Ryzen 5 CPU, 32 GB Ram and an RTX 3060 GPU with 12 GB VRam. To date, I've been running the early Fooocus version locally. I do not use Colab.

**Reply by mashb1t** [reply]
It's easiest if you download the zip file from [here](https://github.com/lllyasviel/Fooocus?tab=readme-ov-file#download), unzip it with [7zip](https://7-zip.de/index.html) (no other program, might lead to issues) and double-click the the run.bat.

**Reply by Cyclonejack88** [reply]
I am having the same issue, Downloaded the 7zip from github, decompressed with 7zip, opened the folder to find no run.bat file.
